File: Excel_Programs/Java/question4.java
//Given an array of size N containing only 0s, 1s, and 2s; sort the array in ascending order.
class question4 
{
    public static void main(String[] args) 
    {
        int arr[] = {0,2, 1, 2,0};

        int count0 =0;
        int count1 =0;
        int count2 =0;

        for (int i = 0;i<5;i++)
        {
            if(arr[i]==0)
            count0 ++;
            if(arr[i]==1)
            count1 ++;
            if(arr[i]==2)
            count2 ++;
        }
        //System.out.println(count0+" "+count1+" "+count2);
        for(int j=0;j<count0;j++)
        {
            arr[j]=0;
        }
        for(int j =count0;j<(count0+count1);j++)
        {
            arr[j]=1;
        }
        for(int j=(count0+count1);j<(count0+count1+count2);j++)
        {
            arr[j]=2;
        }
    
        for(int i=0;i<5;i++)
        {
            System.out.print(arr[i]+" ");
        }
        System.out.println();
    }
    
}

File: Excel_Programs/Python/question4.py
#Given an array of size N containing only 0s, 1s, and 2s; sort the array in ascending order.
import numpy as np
arr = np.array([0,2,1,2,0])
count0=0
count1=0
count2=0

for i in range(5):
    if(arr[i]==0):
        count0 +=1
    if(arr[i]==1):
        count1 +=1
    if(arr[i]==2):
        count2 +=1
for i in range(count0):
    arr[i]=0
for i in range(count0,(count0+count1),1):
    arr[i]=1
for i in range((count0+count1),(count0+count1+count2),1):
    arr[i]=2
for i in range(5):
    print(arr[i],end=' ')
print()
